src/parser.js

```diff
@@ -1238,6 +1262,11 @@ export class GenericParser extends Tokenizer {
           rest: null,
         }));
       }
+      case 'AssignmentExpression': {
```

Comment:
Why is this here? This should only be allowed in the `transformDestructuringWithDefault` case, I would think.

Reply:
yes. Forgot to remove it.

Reply:
@Shilpi3 it's still not removed